### PR TITLE
Initial impl

### PR DESCRIPTION
**Pull request by peterschubert**

### CODE REVIEW

File: helm/resilient-splunk-forwarder/values.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  hasHealthcheck: "true"
+replicaCount: 1
+image:
+  repository: coco/resilient-splunk-forwarder
+  version: "" # should be set explicitly at installation
+  pullPolicy: IfNotPresent
+resources:
+  limits:
+    memory: 256Mi
+  request:
```
Comment (tamas-molnar) [inline helm/resilient-splunk-forwarder/values.yaml]: `requests`

File: helm/resilient-splunk-forwarder/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: The resilient-splunk-forwarder chart for Kubernetes
+name: resilient-splunk-forwarder
+version: 0.0.0
```
Comment (SCBbestof) [inline helm/resilient-splunk-forwarder/Chart.yaml]: Shouldn't this be updated?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,87 @@
...
+## Other information
+There is a single thread listing objects from S3, but actual data is fetched asynchronously. Messages are immediately deleted from S3.
+Messages are then dispatched to a set of workers that submit the data to the configured Splunk HEC URL.
+Failed messages are stored again in S3. Failures also cause exponential backoff so that the endopint is not overwhelmed. 
+However, due to having multiple workers, this will not affect messages that are already dispatched.  
+
+### Logging
+
+* The application uses [logrus](https://github.com/Sirupsen/logrus); the log file is initialised in [main.go](main.go).
+* Logging requires an `env` app parameter, for all environments other than `local` logs are written to file.
```
Comment (BerniVarga) [inline README.md]: Does this part still apply?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,87 @@
...
+* When running locally, logs are written to console. If you want to log locally to file, you need to pass in an env parameter that is != `local`.
+* NOTE: `/__build-info` and `/__gtg` endpoints are not logged as they are called every second from varnish/vulcand and this information is not needed in logs/splunk.
```
Comment (BerniVarga) [inline README.md]: resilient-splunk-forwarder logs won't get into Splunk at all...

File: healthcheck_test.go [hunk 5]
```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	//"errors"
```
Comment (BerniVarga) [inline healthcheck_test.go]: Please run a `go fmt` or a `goimports`.

File: LICENSE [hunk 6]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Financial Times
+Copyright (c) 2016-2017 Financial Times
```
Comment (SCBbestof) [inline LICENSE]: Should we update this to 2016-2018 ?

File: healthcheck_test.go (hunk 5 above)
Comment (SCBbestof) [reply healthcheck_test.go]: My bad, did that but it won't remove comments. I'll edit the small changes, just waiting for the rest of the comments to be reviewed.

File: helm/resilient-splunk-forwarder/Chart.yaml (hunk 2 above)
Comment (BerniVarga) [reply helm/resilient-splunk-forwarder/Chart.yaml]: Jenkins will handle the version changes during the deployment.

File: README.md (hunk 3 above)
Comment (SCBbestof) [reply README.md]: Logrus seems to log to stdout in this case. Removed.

**Pull request merged**